<ui version="4.0" >
 <class>Browser</class>
 <widget class="QMainWindow" name="Browser" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>29</y>
     <width>800</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QWebView" name="webView" >
      <property name="url" >
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewWindow" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="goMenu" >
    <property name="title" >
     <string>&amp;Go</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Settings" >
    <property name="title" >
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionConfigure" />
   </widget>
   <addaction name="fileMenu" />
   <addaction name="goMenu" />
   <addaction name="menu_Settings" />
  </widget>
  <action name="actionNewWindow" >
   <property name="text" >
    <string>New &amp;Window</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConfigure" >
   <property name="text" >
    <string>&amp;Configure...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>Browser</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
